Jenkinsfile

```diff
@@ -61,6 +61,14 @@ node ('mongodb-2.4') {
       govuk.pushTag(REPOSITORY, env.BRANCH_NAME, 'release_' + env.BUILD_NUMBER)
     }
 
+    // Push the Go binary for the build to S3, for AWS releases
```

Comment:
Do we actually want this before line 53 ("archive binaries from the build"), otherwise it won't know what the 'router` artefact is?

Reply:
I think the `Archive Artifact` stage creates the artefact, and this stage uploads it to S3, so seems like the correct order to me.

Reply:
Ohhh. I thought it meant "delete/hide", for some reason.